docs/user-guide/sinks/kafka.md

```diff
@@ -1,16 +1,34 @@
 # Kafka Sink
 
+Two methods are available for integrating Kafka topics into your Numaflow pipeline:
+using a user-defined Kafka Sink or opting for the built-in Kafka Sink provided by Numaflow.
+
+## Option 1: User-Defined Kafka Sink
+
+Developed and maintained by the Numaflow contributor community,
+the [Kafka Sink](https://github.com/numaproj-contrib/kafka-java) provides a reliable and feature-complete solution for publishing messages to Kafka topics.
+
+Key Features:
+
+* **Customization:** Offers complete control over Kafka Sink configurations to tailor to specific requirements.
+* **Kafka Java Client Utilization:** Leverages the Kafka Java client for reliable message publishing to Kafka topics.
+* **Schema Management:** Integrates seamlessly with the Confluent Schema Registry to support schema validation and manage schema evolution effectively.
+
+More details on how to use the Kafka Sink can be found [here](https://github.com/numaproj-contrib/kafka-java?tab=readme-ov-file#write-data-to-kafka).
+
+## Option 2: Built-in Kafka Sink
```

Comment:
Reverse the order of the 2 options?

Reply:
we are eventually deprecating builtin, right?

Reply:
I put `kafka-java` first because I want it to get some attentions and feedbacks so that we can further improve it towards fully feature-complete.

Reply:
@whynowy mind if I keep `kafka-java` as first?

Reply:
Go ahead. I assume we will use this one as the builtin soon.

Reply:
Thanks!